lib/ansible/plugins/action/synchronize.py

```diff
@@ -326,7 +328,6 @@ def run(self, tmp=None, task_vars=None):
             private_key = self._play_context.private_key_file
 
             if private_key is not None:
-                private_key = os.path.expanduser(private_key)
```

Comment:
Oh, sorry, got it. Should we have `expanduser` in both places to be "defensive", in case `private_user` is used somewhere else in the module in the future?